src/components/Button/style.scss

```diff
@@ -15,15 +14,15 @@
   cursor: pointer;
   border-radius: 4px;
   font-weight: 600;
-  display: inline-flex;
+  display: inline-block;
   background-color: #fff;
   color: $font-primary-color;
   border-color: $border-primary-color;
   font-size: 15px;
-  padding: 0 16px;
+  padding: 7px 16px;
   min-width: 36px;
   min-height: 36px;
-  line-height: 36px;
+  line-height: 1.3;
```

Comment:
I think we should use `px` here to keep consistency - we use pixels for `line-height` [almost everywhere](https://github.com/livechat/design-system/search?p=3&q=%22line-height%3A%22&unscoped_q=%22line-height%3A%22).

For `1.3` the pixel equivalent would be `19px / 20px`